src/features/locking/useLockedStatus.ts

```diff
@@ -39,24 +40,34 @@ async function fetchLockedStatus(
   publicClient: PublicClient,
   address: Address,
 ): Promise<LockedStatus> {
-  const [totalLockedResp, pendingWithdrawalsResp] = await publicClient.multicall({
-    contracts: [
-      {
-        address: Addresses.LockedGold,
-        abi: lockedGoldABI,
-        functionName: 'getAccountTotalLockedGold',
-        args: [address],
-      } as const,
-      {
-        address: Addresses.LockedGold,
-        abi: lockedGoldABI,
-        functionName: 'getPendingWithdrawals',
-        args: [address],
-      } as const,
-    ],
-  });
-  if (totalLockedResp.status !== 'success' || pendingWithdrawalsResp.status !== 'success') {
-    throw new Error('Error fetching locked balances or pending withdrawals');
+  const [totalLockedResp, pendingWithdrawalsResp, unlockingPeriodResp] =
+    await publicClient.multicall({
+      contracts: [
+        {
+          address: Addresses.LockedGold,
+          abi: lockedGoldABI,
+          functionName: 'getAccountTotalLockedGold',
+          args: [address],
+        } as const,
+        {
+          address: Addresses.LockedGold,
+          abi: lockedGoldABI,
+          functionName: 'getPendingWithdrawals',
+          args: [address],
+        } as const,
+        {
+          address: Addresses.LockedGold,
+          abi: lockedGoldABI,
+          functionName: 'unlockingPeriod',
+        } as const,
+      ],
+    });
+  if (
+    totalLockedResp.status !== 'success' ||
```

Comment:
remember you can use [allowFailure](https://wagmi.sh/core/api/actions/multicall#allowfailure)